src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -622,11 +622,20 @@ export class HaConfigDevicePage extends LitElement {
       import(
         "./device-detail/integration-elements/zwave_js/ha-device-info-zwave_js"
       );
+      import(
+        "./device-detail/integration-elements/zwave_js/ha-device-actions-zwave_js"
+      );
       templates.push(html`
         <ha-device-info-zwave_js
           .hass=${this.hass}
           .device=${device}
         ></ha-device-info-zwave_js>
+        <div class="card-actions" slot="actions">
+          <ha-device-actions-zwave_js
```

Comment:
This is only used once. No need to make this a component. Just render the `<a>` tag here.

Reply:
Planning to add more buttons here in future PRs, should those all go directly in this file directly instead of being a separate component?

Reply:
Oh. This is like one of those integration specific things on the device page 🤔 

No, in that case you can leave it as is.